spec/routing/sessions_routing_spec.rb

```diff
@@ -11,5 +11,9 @@
     it 'routes to #create' do
       expect(post: '/sessions').to route_to('sessions#create')
     end
+
+    it 'routes to #destroy' do
```

Comment:
How about `it { expect(delete: '/sessions').to route_to('sessions#destroy') }`

Reply:
Fixed in 4fd712beb75475b04a2c8ea131f5e87f7ced514c